src/modules/addRemoveEdit.js

```diff
@@ -0,0 +1,78 @@
+import trash from "../assets/icons8-waste-24.png";
+
+const ul = document.querySelector(".list-items");
+let todos = JSON.parse(localStorage.getItem("todos")) || [];
+export const getToDos = () => todos;
+export const setToDos = (newTodos) => {
+  todos = newTodos;
+};
+
+export const addToDo = (todo) => {
+  const x = {
+    description: todo.description,
+    completed: todo.completed,
+    index: todo.index,
+  };
+  todos.push(x);
+  localStorage.setItem("todos", JSON.stringify(todos));
+  return x;
+};
+
+export const removeTodos = (predicate, todos) =>
+  todos.filter(predicate).map((todo, index) => {
+    todo.index = index + 1;
+    return todo;
+  });
+
+export const createToDo = ({ description, completed, index }) => {
+  const li = document.createElement("li");
+  const checkbox = document.createElement("INPUT");
+  checkbox.classList.add("check");
+  checkbox.setAttribute("type", "checkbox");
+  checkbox.checked = completed;
+  const item = document.createElement("span");
+  item.classList.add("content");
+  item.appendChild(document.createTextNode(description));
+  item.contentEditable = true;
+  const removeBtn = document.createElement("input");
+  removeBtn.setAttribute("type", "image");
+  removeBtn.setAttribute("src", trash);
+  removeBtn.classList.add("btn-remove");
+  li.append(checkbox, item, removeBtn);
+
+  ul.appendChild(li);
+  checkbox.addEventListener("change", () => {
+    const todo = todos.find((todo) => todo.index === index);
+
+    if (checkbox.checked) {
+      todo.completed = true;
+    } else {
+      todo.completed = false;
+    }
+    // todo.completed
```

Comment:
Please remove unused code instead of commenting it. Commented code makes the repository dirty